infrastructure/Taskfile.yml

```diff
@@ -223,6 +223,36 @@ tasks:
       cmds:
         - task/scripts/promote-workloads-to-prod-nodes.sh
 
+    cluster:mode:release:
+      deps: [cluster:auth , lagoon:cli:config]
+      desc: "Set the cluster in release mode by adding additional nodes to release critical node pools"
+      cmds:
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name app6
+          --resource-group rg-env-dplplat01
+          --node-count 4
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name system2
+          --resource-group rg-env-dplplat01
+          --node-count 5
+
+    cluster:mode:business:
+      deps: [cluster:auth , lagoon:cli:config]
+      desc: "Set the cluster in release mode by adding additional nodes to release critical node pools"
+      cmds:
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name app6
+          --resource-group rg-env-dplplat01
+          --node-count 2
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name system2
+          --resource-group rg-env-dplplat01
+          --node-count 3
```

Comment:
In keeping with the `reset` name I wondered if node counts in nodepools/resource counts was something that had a default value in Terraform or the like?

If that is the case then I suggest then I wonder if we could run Terraform to reset the cluster to a default state and thus node count.

If not the move along :smile:

Reply:
Terraform is out of sync right now, otherwise that would've been the way to go about it